tests/FSharp.Compiler.ComponentTests/Conformance/LexicalAnalysis/Comments.fs

```diff
@@ -20,7 +20,7 @@ module Comments =
         |> ignore
 
     // This test was automatically generated (moved from FSharpQA suite - Conformance/LexicalAnalysis/Comments)
-    //<Expects status="error" id="FS0010" span="(8,3-8,4)">Unexpected symbol '\*' in implementation file$</Expects>
+    
```

Comment:
@dsyme

Is this now valid F# syntax, and what does it do:
````let a2 = (*)````

Reply:
This is not something we have deep enough control over to worry about the exact thing in the error message.

The changes to the parser mean that standalone `*` is parsed as a valid expression (being part of indexer syntax) unless used as operator name `( * )` etc. but later rejected during type checking - previously this was an outright syntax error.  This affects the order of error messages just a little, but that's part of the territory. 

In more detail

* The text `*)` is tokenized as STAR then RPAREN (this hasn't changed).   Previously the parse stage failed at `*` but now fails at `)`.  For the command-line compiler, because the parse stage failed no checking on `*` is done.

* The text `*` is tokenized as STAR (this hasn't changed).   Previously the parse stage failed but now succeeds, and an error is given in the check phase instead.